packages/react-static/templates/typescript/index.d.ts

```diff
@@ -0,0 +1,4 @@
+// Allow importing images,
+// you can add other formats supported by file-loader
+declare module '*.jpg'
```

Comment:
Perhaps we want to copy the type definitions from create-react-app?

Reply:
Would you please provide the link to that?

I've tried looking at its repo, but couldn't find anything interesting:
https://github.com/facebook/create-react-app